apps/dashboard/src/@/components/blocks/MobileSidebar.tsx

```diff
@@ -15,8 +15,9 @@ export function MobileSidebar(props: {
     <Dialog open={isOpen} onOpenChange={setIsOpen}>
       <DialogTrigger asChild>{props.trigger}</DialogTrigger>
       <DialogContent
-        className="p-4 rounded-t-xl rounded-b-none"
+        className="p-4 rounded-t-xl rounded-b-none z-[9001]"
```

Comment:
this scares me, why do we need the custom z-index?

Reply:
good catch, this was a temp fix. Added proper fix and removed it